api/charms/client.go

```diff
@@ -285,3 +287,34 @@ func convertCharmExtraBindingMap(bindings map[string]string) map[string]charm.Ex
 	}
 	return result
 }
+
+func convertCharmLXDProfile(lxdProfile *params.CharmLXDProfile) *charm.LXDProfile {
```

Comment:
it looks like we need this (something similar) on the apiserver side too.

Reply:
agreed, we should add that later.